src/de/grobox/liberario/fragments/DirectionsFragment.java

```diff
@@ -240,6 +244,23 @@ public void onResume() {
 		displayFavouriteTrips();
 		refreshAutocomplete(false);
 
+		// check if there's a preference for selected products for the current network
+		TransportNetwork current_network = Preferences.getTransportNetwork(getContext());
+		String network_pref = current_network.getId().toString() + "_selected_products";
+		SharedPreferences sp = getContext().getSharedPreferences(Preferences.NETWORK_SETTINGS, 0);
```

Comment:
From the API-Docs:
`mode 	int: Operating mode. Use 0 or MODE_PRIVATE for the default operation.  
File creation mode: the default mode, where the created file can only be accessed by the calling application (or all applications sharing the same user ID). `